app/src/main/java/com/eventyay/organizer/data/faq/FaqRepositoryImpl.java

```diff
@@ -72,9 +68,8 @@ public Completable deleteFaq(long id) {
         }
 
         return faqApi.deleteFaq(id)
-            .doOnComplete(() -> repository
-                .delete(Faq.class, Faq_Table.id.eq(id))
-                .subscribe())
+            .doOnComplete(() -> OrgaRoomDatabase
+                .getInstance(context).faqDao().deleteFaq(id))
```

Comment:
You are not subscribing

Reply:
Changed to this but still not working:

```
faqApi.deleteFaq(id)
            .toObservable()
            .flatMapCompletable(var -> {
                OrgaRoomDatabase
                    .getInstance(context).faqDao().deleteFaq(id);
                return Completable.complete();
            })
            .subscribeOn(Schedulers.io())
            .observeOn(AndroidSchedulers.mainThread());
```

Reply:
Obviously it won't

```
OrgaRoomDatabase
     .getInstance(context).faqDao().deleteFaq(id);
```

This is never executed because it is never subscribed. Return this. Why returning Completable.complete()

Reply:
Had tried that too but it didn't work. Although, on using doOnSubscribe() or doOnComplete() with `DELETE FROM Faq` query it works fine (Deletes all FAQs from database). But doesn't work with the current query. So I think maybe there's some problem with the `DELETE FROM Faq WHERE id = :id` query?

Reply:
I think this is because the FAQ id is different from the id which is used as PrimaryKey

Reply:
Updated. Now it's working.